src/main/java/edu/umich/its/spe/SPEMaster.java

```diff
@@ -245,10 +261,13 @@ public void sendSummaryEmail() {
 
 		String summaryString = spesummary.toString();
 
+		String subjectString = String.format("(%d/%d) %s %s",spesummary.getAdded(),spesummary.getErrors(),
```

Comment:
maybe put some label here for the numbers, like "(Added %d, With Errors %d) %s %s"

Reply:
Will add a comment to the email itself.